From 5ef33815bfecec8c2126fde98a0830816a99d564 Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Fri, 23 Mar 2018 21:45:30 +0100
Subject: [tools/build] Search Xcode whole installation for MacOSX SDKs

With more recent installations of Xcode the platform SDKs are not
located within the SDK subdirectory anymore.
---
 tools/build/src/tools/darwin.jam | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/tools/build/src/tools/darwin.jam b/tools/build/src/tools/darwin.jam
index 6f7be142..3881290b 100644
--- a/tools/build/src/tools/darwin.jam
+++ b/tools/build/src/tools/darwin.jam
@@ -15,7 +15,7 @@ import toolset : flags ;
 import type ;
 import common ;
 import generators ;
-import path : basename ;
+import path : basename glob-tree ;
 import version ;
 import property-set ;
 import regex ;
@@ -368,8 +368,7 @@ local rule init-sdk ( condition * : root ? : version + : version-feature ? )
 local rule init-available-sdk-versions ( condition * : root ? )
 {
     root ?= /Developer ;
-    local sdks-root = $(root)/SDKs ;
-    local sdks = [ GLOB $(sdks-root) : MacOSX*.sdk AppleTVOS*.sdk AppleTVSimulator*.sdk iPhoneOS*.sdk iPhoneSimulator*.sdk ] ;
+    local sdks = [ glob-tree $(root) : MacOSX*.sdk AppleTVOS*.sdk AppleTVSimulator*.sdk iPhoneOS*.sdk iPhoneSimulator*.sdk ] ;
     local result ;
     for local sdk in $(sdks)
     {
-- 
2.14.3

